test/replication-luatest/gh_5272_recovery_triggers_test.lua

```diff
@@ -0,0 +1,85 @@
+local t = require('luatest')
```

Comment:
AFAIK we don't use `gh_####` prefix for tests about features. That prefix is only for bugfixes, no?

Reply:
Yep, you're right. Renamed.